controller/invitation.go

```diff
@@ -63,7 +63,7 @@ func (c *InvitationController) CreateInvite(ctx *app.CreateInviteInvitationConte
 		})
 	}
 
-	err = c.app.InvitationService().Issue(ctx, currentIdentity.ID, ctx.InviteTo, invitations)
+	err = c.app.InvitationService().Issue(ctx, currentIdentity.ID, ctx.InviteTo, ctx.Payload.Links, invitations)
```

Comment:
As mentioned above, I would suggest that the controller checks the `ctx.Payload.Links` and passes the actual or default values to the service